akvo/rsr/spa/app/modules/editor/project-init-handler.js

```diff
@@ -11,10 +11,14 @@ const insertRouteParams = (route, params) => {
   })
   return route
 }
-let prevParams
+
 let sectionIndexPipeline = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
+const SECTION_RESULTS_INDICATORS = 4;
+const SECTION_COUNT = 11;
```

Comment:
It should be worth changing this to something like

```js
const SECTION_RESULTS_INDICATORS = 4;

let sectionIndexPipeline = [1, 2, 3, SECTION_RESULTS_INDICATORS, 5, 6, 7, 8, 9, 10, 11];
const SECTION_COUNT = sectionindexPipeline.length + 1;
```

If you know the name of the sections, then you could just insert them all for later use.